#66356: Lodestar: Home page: Full width blocks are not full width

Gustavo-Hilario opened the pull request:
#### Changes proposed in this Pull Request:

Removing a CSS selector that was preventing full-width blocks from being full width on the Lodestar home page.

---
<table>
    <thead>
      <tr>
        <th>Before</th>
        <th>After</th>
    </tr>
    </thead>
    <tbody>
      <tr>
        <td><img src="https://user-images.githubusercontent.com/50875131/183492690-1e87f7ae-27ec-4e9d-a801-5264447aacbe.png"/></td>
        <td><img src="https://user-images.githubusercontent.com/50875131/183492699-bf74e276-a165-4ac9-96f5-4435ed17b842.png"/></td>
      </tr>
    </tbody>
  </table>

#### Related issue(s):

Lodestar: Home page: Full width blocks are not full width Automattic/themes#6352

Fixes: https://github.com/Automattic/themes/issues/6352

Gustavo-Hilario committed:
#66356: Lodestart: Home page: Full width blocks are not full width

Adding an extra CSS selector to ensure that Lodestar home page can use full-width blocks when the panel does not split into two columns.

Gustavo-Hilario committed:
#66356: Lodestart: Home page: Full width blocks are not full width

Removing a CSS selector that was preventing full-width blocks from being full width on the Lodestar home page.

pbking commented:
I see that the bit removed was originally added [here on purpose](https://github.com/Automattic/themes/pull/326).  Does the original issue (wide and full elements in two-column layouts) still apply?  Is it appropriately dealt with in another way?

pbking commented:
Yup, can confirm that the original reason that was applied is still valid:
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/146530/183666681-07246a78-e476-471b-b0c3-a09d676079f2.png">

It only seems to matter if it's using 2 columns so maybe something specific to the `.lodestar-panel two-column` classes (or something) is necessary to remain.

Gustavo-Hilario committed:
Merge remote-tracking branch 'origin/fix/#66356' into fix/#66356

Gustavo-Hilario committed:
#66356: Lodestar: Home page: Full width blocks are not full width

Adding an extra CSS selector to ensure that Lodestar home page can use full-width blocks when the panel does not split into two columns.

Gustavo-Hilario commented:
Thank you for the heads-up here, @pbking .

The code was allowing full-width content on all Lodestar panels, except the home page panel that has the `lodestar-intro` class, not `lodestar-panel`. I've added the code that will help with full-width content on the Lodestar home page.

pbking reviewed (approved):
👍 Works as intended with no side effects I could discern.
